registry/access_control/rbac/auth.py

```diff
@@ -115,14 +115,14 @@ def _decode_token(self, token: str) -> Mapping:
                                 'RS256'], audience=config.RBAC_API_AUDIENCE)
             return decode
         except ExpiredSignatureError as e:
-            logging.debug(f'The token signature has expired: {e}')
+            logging.error(f'The token signature has expired: {e}')
```

Comment:
ExpiredSignatureError is client error, should use logging.info instead on server side.